plugins/mac-virtualcam/src/obs-plugin/plugin-main.mm

```diff
@@ -306,7 +306,12 @@ static bool virtualcam_output_start(void *data)
                                                delegate.lastErrorMessage]
                         .UTF8String);
             } else {
-                obs_output_set_last_error(vcam->output, obs_module_text("Error.SystemExtension.NotInstalled"));
+                if (@available(macOS 15.0, *)) {
```

Comment:
I could be wrong but I think this will only ever be true once we compile against the macOS 15 platform SDK.

We'll probably upgrade the used Xcode version on CI as well as the platform SDK once they become available on GitHub Actions after macOS 15 is about to release.

Reply:
This feedback still needs to be resolved.

Reply:
I just checked a CI build in another branch with an `@available(macOS 15.0, *)` check built on Xcode 15.2, and it evaluated to true on my macOS 15 system. So this check does not actually require the macOS 15 SDK to function as intended. I'm comfortable resolving this given that info.